src/util.c

```diff
@@ -1373,6 +1373,13 @@ nats_ReadFile(natsBuffer **buffer, int initBufSize, const char *fn)
     return NATS_UPDATE_ERR_STACK(s);
 }
 
+void nats_FreeAddrInfo(struct addrinfo *res)
```

Comment:
To follow the code style and fix the issue reported on compile, I would write like this:
```
void
nats_FreeAddrInfo(struct addrinfo *res)
{
    // Calling freeaddrinfo(NULL) is undefined behaviour.
    if (res == NULL)
        return;

    freeaddrinfo(res);
}
```